client/main.js

```diff
@@ -0,0 +1,496 @@
+      response.forEach(element => {
+        if (element.status === false) {
+          $(`#Todos-Not-Done`).append(`
+          <div class="card" style="margin: 10px;">
+            <div class="card-header"> 
+            Not Done 
+            </div>
+            <div class="card-body">
+                <h3 class="card-title"> ${element.title} </h3>
+                <p class="card-text"> ${element.description} </p> 
+                <br>
+                <p class="card-text"> ${element.due_date.slice(0,10)} </p>
+                <br>
+                <a href="#" class="btn btn-outline-primary" onclick="DoneTodo(${element.id})"> Done </a>
+                <a href="#" class="btn btn-outline-secondary" onclick="PageUpdateTodo(${element.id})"> Update </a>
+                <a href="#" class="btn btn-outline-danger" onclick="DeleteTodo(${element.id})"> Delete </a>
+              </div>
+            </div>
+          </div>`)
+        } else if (element.status === true) {
+          $(`#Todos-Done`).append(`
+          <div class="card" style="margin: 10px;">
+            <div class="card-header">
+            Done
+            </div>
+            <div class="card-body">
+              <h3 class="card-title"> ${element.title} </h3>
+              <p class="card-text"> ${element.description} </p> 
+              <br>
+              <p class="card-text"> ${element.due_date.slice(0,10)} </p>
+              <br>
+              <a href="#" class="btn btn-outline-primary" onclick="NotDoneTodo(${element.id})"> Not Done </a>
+              <a href="#" class="btn btn-outline-danger" onclick="DeleteTodo(${element.id})"> Delete </a
+            </div>
+          </div>`)
+        }
+      })
+    })
+    .fail( err => {
+      console.log(err);
+    })
+    .always( () => {
+    })
+}
+
+function PageUpdateTodo(id) {
+  $(`#updateTodo-page`).empty()
+  showFormUpdate() 
+  $.ajax({
+    method: `GET`,
+    url: baseUrl + `/todos/${id}`,
+    headers: {
+      access_token: localStorage.access_token
+      }
+  })
+    .done( (response) => {
+        $(`#updateTodo-page`).append(`
+          <div class="row mt-4 border rounded">
+          <form action="">
+            <div class="container position-sticky" >
+              <h2>UPDATE TODO</h2>
+          </div>
+            <div class="mb-3">
+              <label class="form-label">Title </label>
+              <input type="text" class="form-control" id="updateTodo-Title" value="${response.title}">
+            </div>
+            <div class="mb-3">
+              <label class="form-label">Description</label>
+              <input type="textarea" class="form-control" id="updateTodo-Description" rows="3" value="${response.description}">
+            </div>
+            <div class="mb-3">
+              <label class="form-label">Status</label>
+              <input class="form-control" type="text" value="Not Done" id="updateTodo-Status" aria-label="Disabled input example" disabled>
```

Comment:
Mungkin ini kebawa dari create ya yang belom boleh dibikin done
```suggestion
              <input class="form-control" type="text" value="Not Done" id="updateTodo-Status" aria-label="Disabled input example">
```